src/couch_mrview_http.erl

```diff
@@ -433,6 +443,7 @@ parse_params(Props, Keys) ->
 
 parse_params(Props, Keys, #mrargs{}=Args0) ->
     Args = Args0#mrargs{keys=Keys},
+    [check_compatible(K1, K2, Props) || {K1, K2} <- [{"group", "group_level"}]],
```

Comment:
For current `parse_params` implementation, it's better to have such check there. Otherwise we have two places that validates params. Also, these params are not incompatible, but just overlapping: they both sets the same record field, but with different values. Like if you pass `group_level=1&group_level=2`.